homeassistant/components/input_boolean/__init__.py

```diff
@@ -150,18 +150,25 @@ async def reload_service_handler(service_call: ServiceCall) -> None:
     return True
 
 
-class InputBoolean(ToggleEntity, RestoreEntity):
+class InputBoolean(collection.CollectionEntity, ToggleEntity, RestoreEntity):
     """Representation of a boolean input."""
 
     _attr_should_poll = False
+    editable: bool
 
     def __init__(self, config: ConfigType) -> None:
         """Initialize a boolean input."""
         self._config = config
-        self.editable = True
         self._attr_is_on = config.get(CONF_INITIAL, False)
         self._attr_unique_id = config[CONF_ID]
 
+    @classmethod
+    def from_storage(cls, config: ConfigType) -> InputBoolean:
+        """Return entity instance initialized from yaml storage."""
```

Comment:
docstring (also check from_yaml below)
```suggestion
        """Return entity instance initialized from storage."""
```